DataFormats/FEDRawData/src/classes_def.xml

```diff
@@ -24,4 +27,5 @@
  </class>
  <class name="edm::Wrapper<FEDRawDataCollection>" splitLevel="0"/>
  <class name="edm::RefProd<FEDRawDataCollection>"/>
+ <class name="edm::Wrapper<BSTRecord>" splitLevel="0"/>
```

Comment:
@slava77 a memory-refresh question: the `splitLevel` attribute refers to the possibility to split the branch storing the `edm::Wrapper<BSTRecord>` instances. In particular, `splitLevel="0"` is sufficient to force a branch to not be split. In this case, it is in principle a no-op, since the `edm::Wrapper<BSTRecord>` should not create leaves.
Is this correct?